source/platform/renderers/DirectX11/DirectX11.cpp

```diff
@@ -1006,15 +1063,15 @@ Gwk::Color DirectX11::PixelColor(Gwk::Texture* pTexture, unsigned int x, unsigne
 //    for (UINT driverTypeIndex = 0; driverTypeIndex < numDriverTypes; driverTypeIndex++)
 //    {
 //        D3D_DRIVER_TYPE driverType = driverTypes[driverTypeIndex];
```

Comment:
Maybe we could comment this bit at the end, about why it is here, and commented out, and why experimental? TBH I haven't looked at it too closely and I'm not overly familiar with DX11. Perhaps a todo list for someone to fix it if it is broken or needs work? Perhaps we could also add an issue to refer to it so the work is more visible?

Reply:
From what I understood, this is a remnant of GWEN. I developed this renderer from the model of DirectX 9 renderer at the time in GWEN, and it included these functions. These functions handles setup & rendering of a DirectX instance for that window, given a Gwk::WindowProvider. 

It does the some of the work of the DirectX11Sample.cpp

Reply:
Ah ok, I'll have to look into Window Provider. I'm not quite sure why that is there. I think it might be to help setting up an application: common code. I might scrap it to simplify things, so the setup is done in the sample/client app.